cypress/integration/SignIn/signIn.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { formSelectors } from './selectors'
+
+describe('Sign In Page', () => {
+  context('given valid user credential', () => {
+    beforeEach(() => {
+      cy.mockSignInResponse(200, 'signInResponse/valid_response.json')
+      cy.signIn(Cypress.env('CYPRESS_VALID_EMAIL'), Cypress.env('CYPRESS_VALID_PASSWORD'))
```

Comment:
@gutakk Not sure about having this on `beforeEach` regarding the [testing approach](https://nimblehq.co/compass/development/code-conventions/rspec/#fixtures-isolation) we used in `Rspec` that `test must create its own fixtures and setup`

Reply:
Agree with you. Fixed in ad0dd6670ae0602343269af400df61285354732e